tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -153,7 +194,7 @@ FilterPipeline::get_var_chunk_sizes(
 }
 
 Status FilterPipeline::filter_chunks_forward(
-    const WriterTile& tile,
+    WriterTile& tile,
```

Comment:
You should revert this change as well since we don't need to modify the tile any more.

Reply:
Done, thanks!